officemanager/department/migrations/0002_auto_20190904_1052.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.4 on 2019-09-04 10:39
+
+from django.db import migrations
+
+def add_titles_dep(apps, schema_editor):
```

Comment:
There is more efficient way for inserting multiple records into a table - bulk_create: https://docs.djangoproject.com/en/2.2/ref/models/querysets/#bulk-create